R/aaa.R

```diff
@@ -175,6 +175,10 @@ has_validation_fn <- function(f){
 }
 
 validate <- function(handler, ret, dots, label) {
+  # error if label is length > 1
+  if(is.character(label) && length(label) > 1)
+    declare_time_error("Please provide only one label.", declaration)
```

Comment:
s/declaration/ret/

Reply:
thanks corrected. 

but validate() isn't being called for some reason on any step (or at least that code is not being run) i.e.:

`declare_population(N = 5, label = c("lbl1", "lbl2"))`

does not result in error